# This patch ensures executables and libraries are copied in sane folders
# Author: Maxime Boissonneault
diff -ru elastix-5.0.1.orig/CMakeLists.txt elastix-5.0.1/CMakeLists.txt
--- elastix-5.0.1.orig/CMakeLists.txt	2020-09-27 11:18:40.000000000 +0000
+++ elastix-5.0.1/CMakeLists.txt	2021-08-06 13:41:10.686838928 +0000
@@ -135,7 +137,7 @@
 # include in their programs to borrow elastix functionality.
 
 if( NOT LIBRARY_OUTPUT_PATH )
-  set( LIBRARY_OUTPUT_PATH ${elastix_BINARY_DIR}/bin CACHE INTERNAL
+  set( LIBRARY_OUTPUT_PATH ${elastix_BINARY_DIR}/lib CACHE INTERNAL
     "Single output directory for building all libraries." )
 endif()
 
diff -ru elastix-5.0.1.orig/Core/CMakeLists.txt elastix-5.0.1/Core/CMakeLists.txt
--- elastix-5.0.1.orig/Core/CMakeLists.txt	2020-09-27 11:18:40.000000000 +0000
+++ elastix-5.0.1/Core/CMakeLists.txt	2021-08-06 13:49:48.782203415 +0000
@@ -217,7 +217,7 @@
     PROPERTIES INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib:${ITK_DIR}" )
 endif()
 
-install( TARGETS elastix_lib transformix_lib elxCore
+install( TARGETS elastix_lib transformix_lib elxCore elastix_exe transformix_exe
   ARCHIVE DESTINATION ${ELASTIX_ARCHIVE_DIR}
   LIBRARY DESTINATION ${ELASTIX_LIBRARY_DIR}
   RUNTIME DESTINATION ${ELASTIX_RUNTIME_DIR}
